weekly_report.yaml

```diff
@@ -16,4 +16,4 @@ teams:
         Filename: KUBESAW_Report.pdf
     jira_projects: issues.redhat.com/KUBESAW
 config:
-  # There are no tool configuration options, currently.
+  {}  # There are no tool configuration options, currently.
```

Comment:
Would not it make sense to delete `config` object from the configuration if it is not used for anything?

Reply:
Yes (and, in fact, the test configurations omit the key), but it seemed like keeping it as a placeholder was useful as a kind of documentation function.  However, the comment is rather misleading now (there _are_ options **available**, under the `"google"` subsection, but they aren't shown here).

Rather than fix the comment I think I'll take your suggestion and just delete the key.